turbopack/crates/turbo-tasks-backend/src/backend/storage.rs

```diff
@@ -431,9 +465,15 @@ macro_rules! iter_many {
 }
 
 macro_rules! get_many {
-    ($task:ident, $key:ident $input:tt => $value:ident) => {
+    ($task:ident, $key:ident $input:tt => $value:expr) => {
         $crate::backend::storage::iter_many!($task, $key $input => $value).collect()
     };
+    ($task:ident, $key:ident $input:tt => $value:expr) => {
+        $crate::backend::storage::iter_many!($task, $key $input => $value).collect()
+    };
+    ($task:ident, $key:ident $input:tt if $cond:expr => $value:expr) => {
+        $crate::backend::storage::iter_many!($task, $key $input if $cond => $value).collect()
+    };
```

Comment:
This macro can be simplified with the "tt bundling" pattern: https://veykril.github.io/tlborm/decl-macros/patterns/tt-bundling.html

See #71061